include/loops/loops.hpp

```diff
@@ -214,28 +218,12 @@ class Func
     Func* impl;
 };
 
-class Backend
-{
-    friend Backend* _getImpl(Backend* wrapper);
-public:
-    Backend(const Backend& f);
-    Backend& operator=(const Backend& f);
-    virtual ~Backend();
-    virtual void* compile(Context* a_ctx, Func* a_func) const;
-    static Backend makeAarch64Compiler();
-protected:
-    Backend();
-private:
-    Backend(Backend* a_p);
-    Backend* impl;
-};
-
 class Context
 {
     friend Context* _getImpl(Context* wrapper);
 public:
-    Context();
-    Context(Backend cmpl);
+    enum {CF_SPILLSTRESS = 1};
```

Comment:
if you introduce enumeration value for a special mode, you should also have a value for the default mode, such as CF_DEFAULT or something like that

Reply:
Removed. Spill stress control moved to Backend.